packages/ckeditor5-ui/tests/colorpicker/colorpickerview.js

```diff
@@ -50,31 +50,179 @@ describe( 'ColorPickerView', () => {
 
 			clock.tick( 200 );
 
-			expect( view.input.fieldView.value ).to.equal( '#0000ff' );
+			expect( view.inputRow.children.get( 1 ).fieldView.value ).to.equal( '0000ff' );
 		} );
 
 		it( 'should update color property after getting an input', () => {
-			view.input.fieldView.value = '#ff0000';
-			view.input.fieldView.fire( 'input' );
+			view.inputRow.children.get( 1 ).fieldView.value = '#ff0000';
+			view.inputRow.children.get( 1 ).fieldView.fire( 'input' );
 
 			clock.tick( 200 );
 
 			expect( view.color ).to.equal( '#ff0000' );
 		} );
 
-		// Requires https://github.com/ckeditor/ckeditor5/pull/13819 - text input should be properly registered as a focusable.
-		it( 'should not update color property during input editing when focused', () => {
-			view.color = '#000000';
+		describe( 'should update color property', () => {
+			describe( 'when hex color was passed', () => {
+				describe( 'in 3-character format', () => {
+					it( 'with `#` at the beginning', () => {
```

Comment:
It's nice that despite such a long chain you kept sense in the combined name 🏆

![](https://user-images.githubusercontent.com/5353898/234785853-f5dad0c5-01ea-4d56-93ae-a80c0cda0f42.png)